client/landing/jetpack-cloud/components/log-item/index.tsx

```diff
@@ -35,9 +37,13 @@ class LogItem extends React.PureComponent< Props > {
 	}
 
 	render() {
-		const { highlight, children } = this.props;
+		const { highlight, children, className } = this.props;
 		return (
-			<FoldableCard header={ this.renderHeader() } className="log-item" highlight={ highlight }>
+			<FoldableCard
```

Comment:
Out of curiosity, could you use the `ActivityList` for items like this?

Reply:
Do you mean to use `ActivityList` to wrap the list of ignored and fixed events? In theory, we could but I'm not convinced we should. The UIs of the backup and scan events are not the same, so we would have to add to the `ActivityList` non-trivial logic to work well in both cases.